.env.test

```diff
@@ -4,8 +4,13 @@ FLASK_ENV=development
 FLASK_SKIP_DOTENV=True
 
 # Change the following values for local development.
+LOGIN_DISABLED=False
 SECRET_KEY=secret-key
 API_KEY=api123
 TOKEN=token123
 BOARD_ID=board123
-TODO_LIST_ID=456
\ No newline at end of file
+TODO_LIST_ID=456
+MONGO_DB_CONNECTION=mongodb+srv://rkorol1:Docker123@cluster0.1pfca.mongodb.net/myFirstDatabase?w=majority
+MONGO_DB_DATABASE=Test_Database
+COSMOS_MONGO_DB_CONNECTION=mongodb://cosmosdb-todoapp-account:fLxPh72uvrIu3pMqfD78HyBO9XMA0eA2YTSfBX5wxHwgngQ9AfoVcBgJ9yEYFpF7Xzg7pw5ghnBpEvG55iq4qw==@cosmosdb-todoapp-account.mongo.cosmos.azure.com:10255/DefaultDatabase?ssl=true&replicaSet=globaldb&retrywrites=false&maxIdleTimeMS=120000&appName=@cosmosdb-todoapp-account@
```

Comment:
This and your `MONGO_DB_CONNECTION` look like genuine connection strings; we shouldn't be checking those in to our source control! We should remove these (we can replace them with test stubs, like `COSMOS_MONGO_DB_CONNECTION=fake-connection-string` and treat those passwords as compromised and rotate them
(Note that you're passing the relevant variable through in Travis, so that's how we can inject it safely)